### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by dtusci** [community/community#88019] [Programming Help]
### Select Topic Area

Question

### Body

Hi community!
I’m trying to link the Moodle and PayPal platforms. I do all the steps to configure the bottom to appear in my Moodle site so that people can automatically pay for a given course. However the PayPal button is not working properly because people need to create an account or get authentication in the  Moodle but I want the PayPal method work without creating an account. Do you have any suggestions like need to do some coding or else? I tried to many YouTube videos which seems to explain the steps but something happens because it’s not working in my case. Any suggestions?
Thanks a lot !

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @dtusci ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
